aws-android-sdk-iot-test/src/androidTest/java/com/amazonaws/mobileconnectors/iot/MqttManagerIntegrationTest.java

```diff
@@ -115,13 +101,9 @@ public void afterEach() {
 
     @Test
     public void mqttConnect_USEast1_ATS_Endpoint() throws Exception {
-        DescribeEndpointRequest request = new DescribeEndpointRequest();
-        request.setEndpointType("iot:Data-ATS");
-        DescribeEndpointResult result = iotClient.describeEndpoint(request);
-        endpointPrefix = result.getEndpointAddress().split("\\.")[0];
-        System.out.println("Endpoint prefix: " + endpointPrefix);
-        
-        String endpoint = endpointPrefix + ".iot." + Region.getRegion(Regions.US_EAST_1).getName() + ".amazonaws.com";
+        String prefix = iotClient.getEndpointPrefix("iot:Data-ATS");
+        String region = Regions.US_EAST_1.getName();
+        String endpoint = String.format("%s.iot.%s.amazonaws.com", prefix, region);
         System.out.println("Endpoint: " + endpoint);
```

Comment:
I wonder about these print statements throughout the code base. Why? Maybe if we move the format call into a method, we can call `Log` instead?

Reply:
Yea, `System.out` is a weird thing to have in Android code. Since there are a lot of them, I'll save it for a different PR -- maybe if I do another cleanup PR in this file...